R/combinedscatterplot.R

```diff
@@ -397,8 +416,8 @@ CombinedScatter <- function(x = NULL,
         fit.y = fit$fit.y,
         fit.group = fit$fit.group,
         fit.panel = fit$fit.panel,
-        fit.lower.bound = fit$fit.lower.bound,
-        fit.upper.bound = fit$fit.upper.bound,
+        fit.lower.bound = if (fit.CI.show) fit$fit.lower.bound else NULL,
```

Comment:
I encountered strange issues with using inline `if else` when passing parameters, but I assume it works for you?

Reply:
This worked for me